share/gruvbox_config.py

```diff
@@ -0,0 +1,187 @@
+from barpyrus import hlwm
+from barpyrus import widgets as W
+from barpyrus.core import Theme
+from barpyrus import lemonbar
+from barpyrus import conky
+from barpyrus.colors import (
+    AQUA_LIGHT,
+    GREEN_LIGHT,
+    YELLOW_LIGHT,
+    PURPLE_LIGHT,
+    BLUE_LIGHT,
+    ORANGE_LIGHT,
+    RED_LIGHT,
+    FG,
+    BG,
+    col_fmt,
+)
+import sys
+# Copy this config to ~/.config/barpyrus/config.py
+
+# set up a connection to herbstluftwm in order to get events
+# and in order to call herbstclient commands
+hc = hlwm.connect()
+
+# get the geometry of the monitor
+monitor = sys.argv[1] if len(sys.argv) >= 2 else 0
+(x, y, monitor_w, monitor_h) = hc.monitor_rect(monitor)
+height = 14 # height of the panel
+width = monitor_w # width of the panel
+hc(['pad', str(monitor), str(height)]) # get space for the panel
+
+# Conky setup
+custom = ''
+
+mail_symb = '\ue1a8'
+inbox = '/home/axel/.mail/INBOX'
```

Comment:
I think one should replace this by
```
inbox = os.path.join(os.environ['HOME'], '.mbox/inbox')
```
(also requires `import os` above)

Reply:
I'd vote for `os.path.expanduser('~/.mbox/inbox')` instead :wink:

Reply:
Yes, that's much better!

Reply:
Woops, indeed!

Reply:
Done